mmcv/ops/csrc/common/cuda/riroi_align_rotated_cuda_kernel.cuh

```diff
@@ -0,0 +1,246 @@
+// Modified from
+// https://github.com/csuhan/ReDet/blob/master/mmdet/ops/riroi_align/src/riroi_align_kernel.cu
+#ifndef RIROI_ALIGN_ROTATED_CUDA_KERNEL_CUH
+#define RIROI_ALIGN_ROTATED_CUDA_KERNEL_CUH
+
+#include <float.h>
+#ifdef MMCV_WITH_TRT
+#include "common_cuda_helper.hpp"
```

Comment:
@grimoire , is it necessary to include the `common_cuda_helper.hpp`  here?

Reply:
Nope, we do not plan to add TensorRT support for now.